main.go

```diff
@@ -2,16 +2,18 @@ package main
 
 import (
 	"context"
-	"log"
 	"os"
 	"os/signal"
 
+	"github.com/chainguard-dev/clog"
 	"github.com/wolfi-dev/wolfictl/pkg/cli"
 )
 
 func main() {
-	if err := mainE(context.Background()); err != nil {
-		log.Fatalf("error during command execution: %v", err)
+	ctx := context.Background()
+	if err := mainE(ctx); err != nil {
+		clog.FromContext(ctx).Error(err.Error())
```

Comment:
Just curious why not use Fatal here? It logs with level error, and then exits.

Reply:
Lack of intelligence, mainly!

Reply:
Hardly! 🤣